exp-player/addon/components/isp-radio-group/template.hbs

```diff
@@ -2,21 +2,37 @@
     {{#each options as |option index|}}
       <label class="text-center label-text {{if formatLabel formatLabel}}">
           {{#if labelTop}}
-            {{t option}}
+            {{#if hiddenOptions}}
+              {{#each hiddenOptions as |hidden|}}
+                <span class="{{if (eq hidden option) 'hidden'}}">{{t option}}</span>
+              {{/each}}
+            {{else}}
+                {{t option}}
+            {{/if}}
             {{radio-button value=(last-segment option) checked=value}}
             {{#each-in labels as |ratings value|}}
               {{#if (eq value.rating option)}}
-                {{t value.label}}
+                <span class="{{if value.formatClass value.formatClass}}">
+                    {{t value.label}}
+                </span>
               {{/if}}
             {{/each-in}}
           {{else}}
             {{#each-in labels as |ratings value|}}
               {{#if (eq value.rating option)}}
-                {{t value.label}}
+                <span class="{{if value.formatClass value.formatClass}}">
+                    {{t value.label}}
+                </span>
               {{/if}}
             {{/each-in}}
             {{radio-button value=(last-segment option) checked=value}}
-            {{t option}}
+            {{#if hiddenOptions}}
```

Comment:
This code renders spans that contain the label, but with a class to mark it as hidden.

If you don't want the label to be visible, wouldn't it be simpler to not render the label at all in this block? Line 31 is sending sort of a mixed message.